### ISSUE TITLE
pywps sets 444 on output file

### ISSUE THREAD

**Issue by landam** [geopython/pywps#37]
Hi,

PyWPS 3.x changes permission on referenced output file, see [1]. In the result the user who runs the process (usually www-data) cannot delete this file.

```
 -r--r--r-- 2 www-data www-data 57590190 Aug 25 14:22 /tmp/basin_20252/basin.zip
```

Why PyWPS behaves like that? Thanks, Martin

[1] https://github.com/geopython/PyWPS/blob/master/pywps/Wps/Execute/__init__.py#L1135

**Comment by cehbrecht** [comment]
probably that goes on my account ... not sure why i'm explicitly setting the permissions but probably there were reasons ...

I would change the permissions to 644. I've no commit rights so i can i only send a pull request ...

Cheers,
Carsten

**Comment by landam** [comment]
Thanks for quick feedback!

**Comment by tomkralidis** [comment]
Can we close this issue?

**Comment by cehbrecht** [comment]
this is fixed with patch #38 which is already merged.
